transmogrifier/sources/transformer.py

```diff
@@ -26,29 +29,13 @@
 if TYPE_CHECKING:
     from collections.abc import Callable, Iterator
 
-    from transmogrifier.models import TimdexRecord
-
 logger = logging.getLogger(__name__)
 
 type JSON = dict[str, "JSON"] | list["JSON"] | str | int | float | bool | None
 
 PARQUET_DATASET_BATCH_SIZE = 1_000
 
 
-@dataclass
-class ETLRecord:
-    timdex_record_id: str | None
-    serialized_source_record: bytes | None
-    transformed_record: TimdexRecord | None
-    action: str
-
-    def serialized_transformed_record(self) -> bytes | None:
-        """Serialize TimdexRecord to binary JSON string if set."""
-        if self.transformed_record:
-            return json.dumps(self.transformed_record.asdict()).encode()
-        return None
```

Comment:
This was removed in favor of using the `DatasetRecord` as provided by the new `timdex-dataset-api` library.